pkgs/applications/networking/protonvpn-gui/default.nix

```diff
@@ -17,8 +17,6 @@
   proton-vpn-local-agent,
   proton-vpn-logger,
   proton-vpn-network-manager,
-  proton-vpn-network-manager-openvpn,
```

Comment:
Same thing here, please remove the deps that are no longer required in the pvpn-gui update commit.

Reply:
I've squashed all package deprecation into the update commit.

Reply:
That's works too, thanks.

Reply:
Actually, that's better even as there's no inconsistent state between commits.